analysis/test/src/pointers.rs

```diff
@@ -14,9 +14,22 @@ extern "C" {
     fn calloc(_: libc::c_ulong, _: libc::c_ulong) -> *mut libc::c_void;
     fn realloc(_: *mut libc::c_void, _: libc::c_ulong) -> *mut libc::c_void;
     fn free(__ptr: *mut libc::c_void);
+}
+
+#[cfg(not(feature = "miri"))]
+extern "C" {
     fn printf(_: *const libc::c_char, _: ...) -> libc::c_int;
 }
 
+#[cfg(feature = "miri")]
+fn printf(fmt: *const libc::c_char, i: i32) -> libc::c_int {
```

Comment:
please add a comment here similar to the PR description explaining why this is done

Reply:
Fixed in https://github.com/immunant/c2rust/pull/686/commits/c02f3e4f0e9170de80facee19cf3a960e7a27ab8.